sidebars.js

```diff
@@ -32,7 +32,8 @@ module.exports = {
 
     ],
     Advanced: [
-      'advanced/thread_safety'
+      'advanced/thread_safety',
+      'advanced/dev_workflow.md'
```

Comment:
Should be the id, DevWorkflow

Reply:
Ah, I renamed the file to snake_casing wil update the id as wel